content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -118,18 +122,27 @@ The cluster status is updated to **Deleting** while cluster resources are being
 
 <br />
 
-## Force Delete a Cluster
 
-If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scopes. To force delete a cluster, follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). After 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated remaining duration before the force deletion becomes available.
+# Upgrading a MAAS Cluster
 
-<br />
+Upgrading a cluster involves careful planning and executing the upgrade while minimizing disruptions to your applications and services. Spectro Cloud provides an intuitive and streamlined upgrade process through the MAAS, simplifying the entire process.
+
```

Comment:
Address the user, don't use Spectro Cloud,  as it separates us from the reader. 
The other statement, "Spectro Cloud provides an intuitive and streamlined upgrade process through the MAAS, simplifying the entire process."  Remove this, unless you have gone through the process and can validate it's simple. Otherwise, it's a frustrating statement to read. This is what we talked about in the last team meeting about avoid terms like simple, easy, etc.

Reply:
Yes, Karl. This was something extra, this wasn't supposed to be there. My bad.